library/src/commonMain/kotlin/com/mercury/sqkon/db/QueryExt.kt

```diff
@@ -17,7 +17,18 @@ data class Eq<T : Any>(
             parameters = 2,
             bindArgs = {
                 bindString(builder.buildPath())
-                bindString(value)
+                when (value) {
+                    is Boolean -> bindBoolean(value)
+                    is ByteArray -> bindBytes(value)
+                    is Double -> bindDouble(value)
+                    is Number -> bindLong(value.toLong())
+                    is String -> bindString(value)
+                    null -> bindString(null)
```

Comment:
do mine eyes deceive me, or does this fix #9 ?

Reply:
Ahh nice, yes I missed that issue, was an oversight on my behalf when creating